### PR TITLE
Exercise 3 text: background information

### PR DESCRIPTION
**Pull request by annagav**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,6 +58,3 @@ target/
 .idea/*
 *~
 scbU
-npm-debug.log
-.DS_Store
-html_app/index.html
```
Comment (carsongee) [inline .gitignore]: why did you remove these from .gitignore?

File: .gitignore [hunk 2]
```diff
@@ -58,10 +58,12 @@ target/
...
+
 npm-debug.log
 .DS_Store
 html_app/index.html
 
-
 .vagrant/
 static/
+
+
```
Comment (pdpinch) [inline .gitignore:69]: It would be nice to clean up these blank lines, but I don't think it's worth the rebase right now.

File: backend/models.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-from django.db import models
 from django.contrib import admin
 from django.forms import ModelForm
```
Comment (pdpinch) [inline backend/models.py:3]: Just confirming: this removal is because the import was redundant?

```
from django.db import models
```

File: html_app/components/FACSModelFactory.js [hunk 4]
```diff
@@ -101,10 +101,10 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
             function s_block(x) {
 				return Math.exp(-((2 - x) * Math.exp(2 - x) - .9) * ((2 - x) * Math.exp(2 - x) - .9) / .4);
             }
-//            function s_block_C(x){
-//                return Math.exp(-((1.3 - x) * Math.exp(1.3 - x) - .9) * ((1.3 - x) * Math.exp(1.3 - x) - .9)/ 0.4);
-//            }
             function s_block_C(x){
+                return Math.exp(-((0.8 - x) * Math.exp(1 - x) - .9) * ((0.8 - x) * Math.exp(1 - x) - .9)/ 0.4);
+            }
+            /*function s_block_C(x){
```
Comment (pdpinch) [inline html_app/components/FACSModelFactory.js:107]: Not a merge blocker, but going forward you should avoid committing commented code.

File: html_app/components/FACSModelFactory.js [hunk 5]
```diff
@@ -274,11 +273,11 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
                     //only for exercise 2
                     transform:  function(v) {
 //                        return (v>100?Math.log(v+0.0001)/Math.LN10:v);
-
+                        console.log("x_trans="+Math.log(v+0.0001)/Math.LN10);
```
Comment (pdpinch) [inline html_app/components/FACSModelFactory.js:276]: Is this debugging code?

File: backend/models.py (hunk 3 above)
Comment (annagav) [inline backend/models.py:3]: This import was repeated few lines below.

**Comment by pdpinch** [comment]
:+1:

**Comment by bdero** [comment]
Was this merge on purpose? It looks like there are several comments not addressed yet.
